### ISSUE TITLE
Some small things for wget

### ISSUE THREAD

**Issue by dmacks** [fink/fink-distributions#19]
1. There are some obsolete items in the user-facing packaging notes of wget-1.19.2-2:
- DescDetail talks about upstream versions 1.13.x, and especially mentions use of fink's openssl vs apple's because apple's is missing a symbol (vs now being missing altogether)--see also DescPort.
- DescUsage talks about certificate support on OS X 10.4 and 10.5, and nothing else. Are these instructions still valid on 10.9 and beyond?

2. I filed an upstream bug-report about the format-string warning in src/utils.c line 2822. Not critical to fix (just in a diagnostic output). The message is not appearing in their list archives yet:
```
-            logprintf (LOG_VERBOSE, _ ("Skipping key with wrong size (%d/%d): %s\n"),                  
+            logprintf (LOG_VERBOSE, _ ("Skipping key with wrong size (%lu/%d): %s\n"),                  
                        (strlen (begin_pos + 8) * 3) / 4, SHA256_DIGEST_SIZE,                            
                        quote (begin_pos + 8));                                                          
```

Update: see [bug-wget message](http://lists.gnu.org/archive/html/bug-wget/2017-12/msg00022.html)

3. It currently uses openssl100. You can probably hold off testing with openssl110...upstream says at best this release only builds when openssl110 is built with certain options. But if you want to, they do have a [recent git commit to support it](https://gitlab.com/gnuwget/wget/commit/6aa6b669efbef62f60471c18e7fdef0206f92337)  (or could just wait for next release:)

**Comment by akhansen** [comment]
I think the certificate option may still hold true.

**Comment by akhansen** [comment]
Updated in commit  1704410

**Comment by remkos** [comment]
As far as I know, the following issues with wget were covered:

1. Regarding openssl versus Apple's: @akhansen already removed the reference to Apple before I took over as maintainer. See commit 1704410. Now two SSL options are available: openssl and gnutls.
Regarding the certificates: That too @akhansen already modified, removing the reference to 10.4 and 10.5 by commit 1704410.

2. This bug was dealt with in a different way upstream: adding `(int)` in front of `(strlen`.

3. The issue with openssl110 was also tackled upstream by a [commit to support it](https://gitlab.com/gnuwget/wget/commit/6aa6b669efbef62f60471c18e7fdef0206f92337) on 26 Nov 2017. It has been working since wget 1.19.5, at least.

Thus, I think, all matters are closed.

**Comment by dmacks** [comment]
Thanks for double-checking!

**Comment by remkos** [comment]
You're welcome. Was nice to do some code archeology.
With this issue closed, maybe you can approve PR #345.
